doc/_static/css/common.css

```diff
@@ -26,3 +26,7 @@
 .tfm {
     background-color: #2980b9;
 }
+
+.matter {
+    background-color: #97DBE1;
```

Comment:
I am a little colourblind, but this color is barely visible for me (e.g. in the left menu) - too bright. What about one proposed by @greg-fer previously - #6D42C8

Reply:
Yes, the current color looks OK on the dropdown menu, but makes page titles in the hierarchy menu hard to read.

Reply:
done